sphinx_design/dropdown.py

```diff
@@ -142,7 +142,11 @@ class DropdownHtmlTransform(SphinxPostTransform):
 
     def run(self):
         """Run the transform"""
-        for node in self.document.traverse(lambda node: is_component(node, "dropdown")):
+        # Can just use "findall" once docutils 0.18+ is required
```

Comment:
the change was introduced in 0.18.1
```suggestion
        # Can just use "findall" once docutils 0.18.1+ is required
```